onnxscript/rewriter/pattern.py

```diff
@@ -169,28 +170,43 @@ class OpPatternBuilder:
 
     def __init__(
         self,
-        opset_pattern: Pattern[str],
-        op_name_pattern: Pattern[str],
+        opset_pattern: OpsetPatternBuilder,
+        op_name: str | Pattern[str],
     ) -> None:
         self.opset_pattern = opset_pattern
-        self.op_name_pattern = op_name_pattern
+        self.op_name = op_name
 
     def __call__(self, *args, **kwargs):
-        # TODO(rama): Unify with convention used elsewhere.
         if "_num_outputs" in kwargs:
-            num_outputs = kwargs["_num_outputs"]
-            del kwargs["_num_outputs"]
+            raise ValueError("Use 'outputs' instead of '_num_outputs'.")
```

Comment:
Is this for future usage? I don't see outputs is used in any case?

Reply:
It is used in the examples/test-cases in generic_pattern_test.